src/Sylius/Bundle/UserBundle/Resources/translations/messages.en.yml

```diff
@@ -1,9 +1,25 @@
 sylius:
-    user:
+    ui:
```

Comment:
The translations keys will be all refactored and all UI related general phrases will be under `ui`, so we do not have to translate `id` 100+ times (for all entities). Discussed with @pjedrzejewski.